File: app/src/main/java/artwork/java/TaskManager.java
package artwork.java;

import artwork.java.R.drawable;

public  class TaskManager {
    int[] taskPics = {drawable.image31, drawable.image4, drawable.romb, drawable.image5, drawable.image6};
    String[] taskData = {"image31.svg", "image4.svg", "romb.svg", "image5.svg", "image6.svg"};
    int currTask = 0;

    public void nextTask(int dtask, MainActivity activity) {
        int taskId = this.currTask + dtask;
        if (taskId != -1 && taskId < this.taskData.length) {
            this.currTask = taskId;
            activity.updateTask(new Task(this.taskData[this.currTask], this.taskPics[this.currTask]),
                    taskData.length, currTask);
        }
    }
}

File: app/src/main/java/artwork/kotlin/TaskManager.kt
package artwork.kotlin

import artwork.java.R

class TaskManager {
    val taskPics: IntArray = intArrayOf(R.drawable.image31, R.drawable.image4, R.drawable.romb,
        R.drawable.image5, R.drawable.image6)
    val taskData: Array<String> = arrayOf("image31.svg", "image4.svg", "romb.svg", "image5.svg", "image6.svg")
    var currTask: Int = 0
    fun nextTask(dtask : Int, activity: MainActivity){
        val taskId = currTask + dtask;
        if (taskId!=-1 && taskId<taskData.size){
            currTask = taskId
            activity.updateTask(Task(taskData[currTask], taskPics[currTask]), taskData.size, currTask)
        }
    }
}
